### PR TITLE
Viite 2383 calibration points in project

### PR DESCRIPTION
**Pull request by skosonen**

### CODE REVIEW

**Comment by skosonen** [comment]
Tests pass: http://livibuild04.vally.local/jenkins/job/viite/job/Run-Tests/305/console

**Comment by skosonen** [comment]
Tests pass: http://livibuild04.vally.local/jenkins/job/viite/job/Run-Tests/317/console
Also in the manual testing the junction point calibration points are respeced in the address calculations at least in unchanged operation.

**Comment by skosonen** [comment]
Tests pass on Jenkins: http://livibuild04.vally.local/jenkins/job/viite/job/Run-Tests/320/console

**Comment by skosonen** [comment]
Includes also VIITE-2393.

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/TrackSectionOrder.scala [hunk 1]
```diff
@@ -125,29 +125,18 @@ object TrackSectionOrder {
   }
 
   def mValueRoundabout(seq: Seq[ProjectLink]): Seq[ProjectLink] = {
-    /* None = No change. Therefore for calibration points
-       None            = No change
+    /* None            = No change. Therefore for calibration points
        Some(None)      = Clear if there was any
        Some(Some(...)) = Set to this value
       */
+    // TODO Check that this works with André's roundabout case
```
Comment (skosonen) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/TrackSectionOrder.scala:132]: Is this verified already?

File: digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/TrackSectionOrder.scala [hunk 2]
```diff
@@ -125,29 +125,18 @@ object TrackSectionOrder {
...
-      val endCP = endCalibrationPoint.getOrElse(pl.calibrationPoints._2) match {
-        case None => Option.empty[ProjectLinkCalibrationPoint]
-        case Some(cp) => Option(CalibrationPointsUtils.toProjectLinkCalibrationPoint(cp, pl.linearLocationId))
-      }
-
-      pl.copy(sideCode = sideCode.getOrElse(pl.sideCode),
-        startAddrMValue = startAddrMValue.getOrElse(pl.startAddrMValue),
-        endAddrMValue = endAddrMValue.getOrElse(pl.endAddrMValue),
-        calibrationPoints = (startCp, endCP)
+      val startCPType = if (pl.originalStartCalibrationPointType == NoCP) RoadAddressCP else pl.startCalibrationPointType
```
Comment (skosonen) [inline digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/TrackSectionOrder.scala:136]: I think this if should also contain check that in this project link the start calibration point is set. Now it adds RoadAddressCP to every project link that has no original calibration point.

Comment (Campanha) [reply digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/process/TrackSectionOrder.scala:136]: yes i believe we dont want to add calibrations to all links, only to those who have it already

**Comment by skosonen** [comment]
Tests pass on Jenkins: http://livibuild04.vally.local/jenkins/job/viite/job/Run-Tests/324/consoleFull

**Pull request merged**